example/pubspec.yaml

```diff
@@ -9,7 +9,8 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  cupertino_icons: ^0.1.3
+  cupertino_icons: ^1.0.3
+  dio: ^4.0.0
```

Comment:
the example files are not meant to run, so don't need dependencies. If you are able to run it and think this will make others life easier, would you mind adding instructions on our README on how to run them?